src/services/ui/src/pages/faq/content/oneMACFAQContent.tsx

```diff
@@ -44,6 +44,29 @@ export const oneMACFAQContent: FAQContent[] = [
   {
     sectionTitle: "General",
     qanda: [
+      {
+        anchorText: "system",
+        question: "Which system should I use for my state’s submission?",
+        answerJSX: (
+          <>
+            <p>
+              Check which system to submit your state plan in with this
+              crosswalk training document.
+            </p>
+            <ul>
+              <li>
+                <a
+                  // href={`${process.env.PUBLIC_URL}/assets/onboarding/eligibility-crosswalk-paper-based-state-plan-macpro.pdf`}
```

Comment:
why is this an env variable?

Reply:
The PDF document that this links out too has a public url that is being stored in secrets manager, this is the same  situation with the video content